packages/components/.storybook/webpack.config.js

```diff
@@ -29,6 +29,7 @@ module.exports = async ({ config, mode }) => {
             alias: {
                 Components: path.resolve(__dirname, '../src/components'),
                 Stories: path.resolve(__dirname, '../stories'),
+                react: path.resolve(__dirname, '../node_modules/react'),
```

Comment:
Why this is added here? :thinking:

Reply:
@reza-fs The `ContractCard` component, uses a custom hook from `@deriv/shared` which has it's own `react`. This causes the `Invalid hook call. Hooks can only be called inside of the body of a function component` error because we had two instances of `react`. So, I added this to fix the error.

Reply:
@ali-fs , PR #1970 has been merged, can you re-test and see if this is still needed?

Reply:
@zaki-binary I removed this line since it is not needed anymore.